management-app/src/main/java/com/sdevelopment/skt/management/controller/IndexController.java

```diff
@@ -0,0 +1,16 @@
+package com.sdevelopment.skt.management.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.Map;
+
+@Controller
+public class IndexController {
+
+    @RequestMapping("/")
+    public String home(Map<String, Object> model) {
```

Comment:
It's not a good practice to literally receive an `Object` object

Reply:
Noted. I used a tutorial where they were using that way of working with Controllers but already updated in ProductController and will do the same here